cms/djangoapps/contentstore/views/videos.py

```diff
@@ -305,6 +311,11 @@ def videos_post(course, request):
             for file in request.json["files"]
     ):
         error = "Request 'files' entry does not contain 'file_name' and 'content_type'"
+    elif any(
+            file['content_type'] not in VIDEO_SUPPORTED_FILE_FORMATS.values()
+            for file in request.json["files"]
```

Comment:
I think, we don't need `request.json.get('files', []) ` because if `files` key is not present, [first condition](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/videos.py#L301) of the if block would be true and this code will never run.

Also, there is already a [test](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/tests/test_videos.py#L197) written to check empty list and this behaviour.